### PR TITLE
[Docs] Add docs for ccall+asyncify. See #13014

### PR DESCRIPTION
**Pull request by kripken**

### CODE REVIEW

File: site/source/docs/porting/asyncify.rst [hunk 1]
```diff
@@ -272,6 +272,23 @@ In this case you don't need to worry about ``ASYNCIFY_IMPORTS``, since it's an
...
 automatically.
 
+Usage with ``ccall``
+####################
+
+When calling a wasm export from JavaScript, if you use ``call`` then you will
+get Promise if it is asynchronous. You can use the Promise normally to get
+notified when the computation completes, and also receive a result if there is
+one. Note that you need to pass ``async: true`` to the ``call`` options object
+for this to work.
```
Comment (curiousdannii) [inline site/source/docs/porting/asyncify.rst:282]: Perhaps this:

> To make use of an Asyncify-using wasm export from Javascript, you can use the ``Module.ccall`` function and pass ``async: true`` to its ``call`` options object. ``ccall`` will then return a Promise, which will resolve with the result of the function once the computation completes.

Comment (kripken) [reply site/source/docs/porting/asyncify.rst]: Thanks! Yes, that's better, updated.

**Comment by ericoporto** [comment]
Hey, your talk about Asyncify could be linked in it's page too: https://youtu.be/qQOP6jqZqf8 (it's really good)

**Comment by kripken** [comment]
Thanks @ericoporto , added!

@sbc100 @RReverser ping for review

**Comment by RReverser** [comment]
I'm not overly familiar with `ccall`, but looks good to me.

**Review by RReverser — APPROVED** [review]

**Pull request merged**